modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -406,6 +406,82 @@ kubectl logs -f <pod-name> --namespace <namespace>
 You can use the output to debug your deployment.
 //end::deployment-pod-pending[]
 
+//tag::deployment-failed-scheduling[]
+=== FailedScheduling
+
+If you see this error, your cluster does not have enough nodes to satisfy the anti-affinity rules:
+
+[.no-copy]
+----
+Warning  FailedScheduling  18m  default-scheduler  0/1 nodes are available: 1 node(s) didn't match pod anti-affinity rules. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod..
```

Comment:
This error can also happen when using volumes that are bound to nodes (NVME for example) as we suggest doing or if your nodes are too small for redpanda's resource requests. (The exact error message may be a touch different. Though it should still be `FailedScheduling`.)

Nodes too small is simple enough to fix but the volume mount issue is much more complicated and is probably better handled by support for the immediate future. Would we want to call out that there are other ways this issue could be encountered so users don't end up at a "dead end" here?